src/libelf.d

```diff
@@ -381,7 +340,7 @@ private:
      *      dictionary
      *      object modules...
      */
-    void WriteLibToBuffer(OutBuffer* libbuf)
+    protected override void WriteLibToBuffer(OutBuffer* libbuf)
```

Comment:
Why protected? It's a final class so nobody can ever call this from a derived class. Would `private override` work as well?